modules/test/playwright/helpers/json-web-services/JSONWebServicesJournalApiHelper.ts

```diff
@@ -130,6 +164,182 @@ export class JSONWebServicesJournalApiHelper {
+		urlSearchParams.append(
+			'ddmStructureId',
+			String(webContent.ddmStructureId)
+		);
+		urlSearchParams.append('ddmTemplateKey', webContent.ddmTemplateKey);
+		urlSearchParams.append(
+			'descriptionMap',
+			JSON.stringify(webContent.descriptionMap)
+		);
+		urlSearchParams.append(
+			'displayDateDay',
+			String(webContent.displayDateDay)
+		);
+		urlSearchParams.append(
+			'displayDateHour',
+			String(webContent.displayDateHour)
+		);
+		urlSearchParams.append(
+			'displayDateMinute',
+			String(webContent.displayDateMinute)
+		);
+		urlSearchParams.append(
+			'displayDateMonth',
+			String(webContent.displayDateMonth)
+		);
+		urlSearchParams.append(
+			'displayDateYear',
+			String(webContent.displayDateYear)
+		);
+		urlSearchParams.append(
+			'expirationDateDay',
+			String(webContent.expirationDateDay)
+		);
+		urlSearchParams.append(
+			'expirationDateHour',
+			String(webContent.expirationDateHour)
+		);
+		urlSearchParams.append(
+			'expirationDateMinute',
+			String(webContent.expirationDateMinute)
+		);
+		urlSearchParams.append(
+			'expirationDateMonth',
+			String(webContent.expirationDateMonth)
+		);
+		urlSearchParams.append(
+			'expirationDateYear',
+			String(webContent.expirationDateYear)
+		);
+		urlSearchParams.append(
+			'externalReferenceCode',
+			webContent.externalReferenceCode
+		);
+		urlSearchParams.append('folderId', String(webContent.folderId));
+		urlSearchParams.append(
+			'friendlyURLMap',
+			JSON.stringify(webContent.friendlyURLMap)
+		);
+		urlSearchParams.append('groupId', String(webContent.groupId));
+		urlSearchParams.append('images', JSON.stringify(webContent.images));
+		urlSearchParams.append('indexable', String(webContent.indexable));
+		urlSearchParams.append('layoutUuid', webContent.layoutUuid);
+		urlSearchParams.append('neverExpire', String(webContent.neverExpire));
+		urlSearchParams.append('neverReview', String(webContent.neverReview));
+		urlSearchParams.append(
+			'reviewDateDay',
+			String(webContent.reviewDateDay)
+		);
+		urlSearchParams.append(
+			'reviewDateHour',
+			String(webContent.reviewDateHour)
+		);
+		urlSearchParams.append(
+			'reviewDateMinute',
+			String(webContent.reviewDateMinute)
+		);
+		urlSearchParams.append(
+			'reviewDateMonth',
+			String(webContent.reviewDateMonth)
+		);
+		urlSearchParams.append(
+			'reviewDateYear',
+			String(webContent.reviewDateYear)
+		);
+		urlSearchParams.append(
+			'serviceContext',
+			JSON.stringify(webContent.serviceContext || {})
+		);
+		urlSearchParams.append(
+			'smallFile',
+			JSON.stringify(webContent.smallFile)
+		);
+		urlSearchParams.append('smallImage', String(webContent.smallImage));
+		urlSearchParams.append('smallImageId', String(webContent.smallImageId));
+		urlSearchParams.append(
+			'smallImageSource',
+			String(webContent.smallImageSource)
+		);
+		urlSearchParams.append('smallImageURL', webContent.smallImageURL);
+		urlSearchParams.append('titleMap', JSON.stringify(webContent.titleMap));
```

Comment:
Could we remove all `String(...` calls?
`urlSearchParams.append` automatically converts values to strings, so we only need to keep the `JSON.stringify(...)` ones.